Packs/Zimperium/Integrations/ZimperiumV2/ZimperiumV2.yml

```diff
@@ -0,0 +1,1294 @@
+      type: String
+    - contextPath: Zimperium.DeviceOsVersion.os.maxOsVersion
+      description: The device operating system max version.
+      type: String
+    - contextPath: Zimperium.DeviceOsVersion.os.name
+      description: The operating system name of the device.
+      type: String
+    - contextPath: Zimperium.DeviceOsVersion.os.osVersionId
+      description: The operating system version identifier of the Device.
+      type: Number
+    - contextPath: Zimperium.DeviceOsVersion.os.patchDate
+      description: The patch date of the device's operating system.
+      type: Date
+    - contextPath: Zimperium.DeviceOsVersion.os.policyCompliant
+      description: Whether the Endpoint's operating system is compliant with the policy.
+      type: Boolean
+    - contextPath: Zimperium.DeviceOsVersion.os.type
+      description: The operating system type.
+      type: String
+    - contextPath: Zimperium.DeviceOsVersion.os.version
+      description: The operating system version.
+      type: String
+    - contextPath: Zimperium.DeviceOsVersion.os.versionUpgradeable
+      description: Whether the device's operating system is upgradeable.
+      type: Boolean
+    - contextPath: Zimperium.DeviceOsVersion.teamId
+      description: The Team ID of the Device.
+      type: String
+    - contextPath: Zimperium.DeviceOsVersion.zdeviceId
+      description: The zdeviceId of the Device.
+      type: String
+  - arguments:
+    - description: The Device id to get CVE's for.
+      name: device_id
+      required: true
+    - description: Maximum number of results to retrieve in each page. Default is 10.
+      name: page_size
+    - defaultValue: '0'
+      description: Page number.
+      name: page
+    - defaultValue: '50'
+      description: Maximum number of results to retrieve in each page. Default is 10.
+      name: limit
+    description: Gets the CVEs associated with a specific device.
+    name: zimperium-cve-devices-get
+    outputs:
+    - contextPath: Zimperium.CVEDevice.id
+      description: The id of the device.
+      type: String
+    - contextPath: Zimperium.CVEDevice.activeExploit
+      description: Whether the CVE is active or not.
+      type: Boolean
+    - contextPath: Zimperium.CVEDevice.exploitPocUrl.exploitPocUrls
+      description: The exploit PocUrls for the CVE.
+      type: Unknown
+    - contextPath: Zimperium.CVEDevice.severity
+      description: The severity of a CVE on a device.
+      type: String
+    - contextPath: Zimperium.CVEDevice.type
+      description: The CVE type.
+      type: String
+    - contextPath: Zimperium.CVEDevice.url
+      description: The URL of the CVE.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      predefined:
+      - EMM
+      - ZIPS
+      description: The module parameter is required to get the groups related to EMM connection or ZIPS connection.
+      name: module
+    description: Get policy groups.
+    name: zimperium-policy-group-list
```

Comment:
what about pagination here?

Reply:
Not supported by the API. I think it is because policy groups will not be so big.